src/function/aggregate/holistic/quantile.cpp

```diff
@@ -16,7 +16,6 @@ namespace duckdb {
 
 // Hugeint arithmetic
 hugeint_t operator*(const hugeint_t &h, const double &d) {
-	D_ASSERT(d >= 0 && d <= 1);
```

Comment:
Perhaps move this into the regular hugeint type, if this is used elsewhere besides only the quantiles?

Reply:
👍

Reply:
Removed this assert without thinking too much about it because it was triggering on some node client tests. However I now failed to reproduce this. Instead of moving this, i now just made the functions static, because moving them to the regular hugeint type was causing other issues.